### PR TITLE
Update Onfinality to Support Multiple Blockchains

### PR DESCRIPTION
**Pull request by GitHubMind**

### CODE REVIEW

File: packages/apps-config/src/endpoints/testing.ts [hunk 1]
```diff
@@ -43,7 +43,8 @@ export const testChains: Omit<EndpointOption, 'teleport'>[] = [
     info: 'aleph-testnet',
     providers: {
       'Aleph Zero Foundation': 'wss://ws.test.azero.dev',
-      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com'
+      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:46]:
```suggestion
      Dwellir: 'wss://aleph-zero-testnet-rpc.dwellir.com'
```

This should make the linter happy.

**Review by jacogr — CHANGES_REQUESTED** [review]
Sorry for the slight (???) delay getting here - made a suggestion to fix the linting issues which should make is merge-able.

File: packages/apps-config/src/endpoints/testing.ts (hunk 1 above)
Comment (GitHubMind) [reply packages/apps-config/src/endpoints/testing.ts:46]: I have submitted pull requests for [9857](https://github.com/polkadot-js/apps/pull/9857) and [9852](https://github.com/polkadot-js/apps/pull/9852). Could you please approve them? Thank you!

**Comment by GitHubMind** [comment]
Hi @jacogr, could you please approve it? It's necessary for me, and it's also urgent. thanks!

**Review by jacogr — APPROVED** [review]

**Pull request merged**